tools/ci_build/github/linux/build_linux_python_package.sh

```diff
@@ -10,14 +10,17 @@ EXTRA_ARG=""
 # Put 3.8 at the last because Ubuntu 20.04 use python 3.8 and we will upload the intermediate build files of this 
 # config to Azure DevOps Artifacts and download them to a Ubuntu 20.04 machine to run the tests.
 PYTHON_EXES=("/opt/python/cp39-cp39/bin/python3.9" "/opt/python/cp310-cp310/bin/python3.10" "/opt/python/cp311-cp311/bin/python3.11" "/opt/python/cp38-cp38/bin/python3.8")
-while getopts "d:p:x:c:" parameter_Option
+while getopts "d:p:x:c:u:" parameter_Option
 do case "${parameter_Option}"
 in
 #GPU or CPU.
 d) BUILD_DEVICE=${OPTARG};;
-p) PYTHON_EXES=(${OPTARG});;
-x) EXTRA_ARG=(${OPTARG});;
+p) PYTHON_EXES=${OPTARG};;
+x) EXTRA_ARG=${OPTARG};;
 c) BUILD_CONFIG=${OPTARG};;
+u) CUDA_VERSION=${OPTARG};;
```

Comment:
You do not need to add an extra parameter here. There is an env var with the same name: “CUDA_VERSION” . You may just use that one.

Reply:
```bash
SHORT_CUDA_VERSION=$(echo $CUDA_VERSION | sed   's/\([[:digit:]]\+\.[[:digit:]]\+\)\.[[:digit:]]\+/\1/')
```
Then later on you can use it as
```
--cuda_version=$SHORT_CUDA_VERSION
```